testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Kafka.java

```diff
@@ -36,11 +42,14 @@
     static final String SCHEMA_REGISTRY = "schema-registry";
     static final String ZOOKEEPER = "zookeeper";
 
+    private final DockerFiles dockerFiles;
```

Comment:
Slightly better approach is to use:
`private final DockerFiles.ResourceProvider configDir;`
and get referrence to all config files only once (typically in the constuctor) via:
`configDir = dockerFiles.getDockerFilesHostDirectory("common/kafka");`

Reply:
Fixed.